library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/WebPImageCompression.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder.base64
+
+import android.graphics.Bitmap
+import android.os.Build
+import android.webkit.MimeTypeMap
+import androidx.annotation.VisibleForTesting
+import androidx.annotation.WorkerThread
+import java.io.ByteArrayOutputStream
+
+/**
+ * Handle webp image compression.
+ */
+class WebPImageCompression internal constructor() : ImageCompression {
+    override fun isOverSizeLimit(bitmapSize: Int): Boolean =
```

Comment:
this should not be part of this class

Reply:
Extracted to a private method in `Base64Serializer`